config.py

```diff
@@ -0,0 +1,38 @@
+# support annotations in older versions of Python
+from __future__ import annotations
+
+import logging
+import os
+from pathlib import Path
+
+# support loading toml files in older versions of Python
+try: import tomllib
+except ModuleNotFoundError: import tomli as tomllib
+
+class Config:
+    def __init__(self, file: str, prefix: str) -> None:
+        self._file = file
+        self._prefix = prefix
+
+    def __load__(self, file: str) -> dict[str, dict] | None:
+        try:
+            with open(Path(__file__).with_name(file), "rb") as config_file:
```

Comment:
Interesting.  2 points.  I would open a .toml file as a text, not binary.  And I usually read toml by file name, instead of these 2 lines I would use `tomllib.load(Path(__file__).with_name(file))`, but maybe I use newer version of the lib that allows it.

Reply:
Doesn't work on the version I'm using; raspberry pis are stuck on an ancient version of Python therefore ancient version of tomlib
```
2024-06-23 20:52:05,205 - ERROR - 'PosixPath' object has no attribute 'read'
Traceback (most recent call last):
```